Sccp_manager.inc/aminterface.class.php

```diff
@@ -428,16 +433,16 @@ function core_sccp_reload()
     }
     function getSCCPVersion()
     {
-        $result = array();
+        //Initialise result array
+        $result = array( 'RevisionHash' => '', 'vCode' => 0, 'RevisionNum' => 0, 'futures' => '', 'Version' => 0);
         if ($this->_connect_state) {
             $_action = new \FreePBX\modules\Sccp_manager\aminterface\SCCPConfigMetaDataAction();
```

Comment:
So happy that SCCPConfigMetaData is actually being used now :-)

Reply:
Note: We can use SCCPConfigMetaData to generate some of the array lists. For example some of the drop downs are based on enums, and SCCPConfigMetaData can provide these. We ought to be able to get SCCPConfigMetaData once during installation or every XX days can cache the json content in a temp file, if speed is a problem. Have a look at what SCCPConfigMetaData contains if you are not familiar.

Reply:
My thoughts exactly - I want to move much of the initialisation to the installer